bluetooth/src/androidLibMain/kotlin/device/DefaultDeviceConnectionManager.kt

```diff
@@ -232,16 +232,16 @@ internal actual class DefaultDeviceConnectionManager(
     }
 
     @SuppressLint("MissingPermission")
-    override suspend fun unpair() {
-        if (device.bondState != BluetoothDevice.BOND_NONE) {
-            deviceWrapper.removeBond()
+    override suspend fun didStartPairing() {
+        if (device.bondState == BluetoothDevice.BOND_NONE) {
+            deviceWrapper.createBond()
         }
     }
 
     @SuppressLint("MissingPermission")
-    override suspend fun pair() {
-        if (device.bondState == BluetoothDevice.BOND_NONE) {
-            deviceWrapper.createBond()
+    override suspend fun didStartUnpairing() {
```

Comment:
Same here, it will not be called when user unpairs device in system settings. And _did_Start goes before any call

Reply:
The point is we need a rename for logging purposes. I've renamed it to did requestStartPairing